target/linux/qualcommax/image/ipq60xx.mk

```diff
@@ -10,3 +10,19 @@ define Device/8devices_mango-dvk
 	DEVICE_PACKAGES := ipq-wifi-8devices_mango
 endef
 TARGET_DEVICES += 8devices_mango-dvk
+
+define Device/yuncore_fap650
+    $(call Device/FitImage)
+    $(call Device/UbiFit)
+    DEVICE_VENDOR := Yuncore
+    DEVICE_MODEL := FAP650
+    BLOCKSIZE := 128k
+    PAGESIZE := 2048
+    DEVICE_DTS_CONFIG := config@cp03-c1
+    SOC := ipq6018
+    DEVICE_PACKAGES := ipq-wifi-yuncore_fap650
+    IMAGES += factory.ubin
```

Comment:
Sorry, but why `factory.ubin`?

Reply:
Please, answer this

Reply:
Is `factory.ubi` also being generated?

Because if it is, then its going to be extra confusing

Reply:
The idea is to replace factory.bin with factory.ubin (can be flashed with stock fw sysupgrade) 
factory.ubi is generated also